api/v1alpha/framework/playwright.schema.json

```diff
@@ -98,6 +98,18 @@
                   "webkit"
                 ]
               },
+              "browserArgs": {
```

Comment:
I think you meant to add this to testcafe. But this file is for playwright :D

Reply:
ah.. updated.